packages/core-flows/src/handlers/customer/find-or-create-customer.ts

```diff
@@ -3,6 +3,7 @@ import { validateEmail } from "@medusajs/utils"
 import { WorkflowArguments } from "@medusajs/workflows-sdk"
 
 type CustomerDTO = {
+  customer?: any
```

Comment:
question: Add type? We could change the DTO name too to better reflect its purpose